ibmcloud-janitor/OWNERS

```diff
@@ -0,0 +1,9 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
not a comment on this file, I would expect these to be under `pkg` directory from top level

Reply:
> not a comment on this file, I would expect these to be under `pkg` directory from top level

Hi @chaodaiG, I'm not sure what you meant here? I don't see a `pkg` directory in the top level !?